nominal/core.py

```diff
@@ -311,6 +339,132 @@ def _from_conjure(cls, client: NominalClient, dataset: scout_catalog.EnrichedDat
         )
 
 
+@dataclass(frozen=True)
+class LogSet:
+    rid: str
+    created_by: str
+    name: str
+    created_at: datetime
+    updated_at: datetime
```

Comment:
since the backend uses nanosecond precision, you'll see that all the types here (annoyingly) have to use IntegralNanosecondsUTC (i.e., int) when it's coming from the backend rather than a nice datetime